### PR TITLE
Update calender.html

### LINKED ISSUE #1019: [Dark Mode] Current date is not visible in the community calendar
### Description
The current date is not visible in the community calendar

Screenshot from 2023-02-13 19-49-10

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by TheOjasSingh**
The background of the current date cell now has a variable color that works in both light and dark modes.
Signed-off-by: Ojas Singh <ojas856@gmail.com>

This PR fixes #1019 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [✅] Yes, I signed my commits.

### CODE REVIEW

File: calendar.html [hunk 1]
```diff
@@ -107,6 +107,10 @@ <h3 style="text-align:center;color:aliceblue;font-size: 1.6rem;">
     animation: load6 1.7s infinite ease, round 1.7s infinite ease;
   }
 
+.fc-unthemed td.fc-today{
+  background: var(--brand-color-primary) !important;
```
Comment (Avi-88) [inline calendar.html:111]: Use --brand-color-secondary instead of primary, it works better with the white text when switched to dark mode

**Review by Avi-88 — COMMENTED** [review]
I've added a change for you to look into , and also when you commit dont forget to sign your commit using the ( -s ) flag. You can checkout the contribution guide to learn more about it. Not signing commits will fail checks in the PR , thanks

**Review by TheOjasSingh — COMMENTED** [review]
@Avi-88 I made the changes you mentioned and signed the commit.

**Comment by Avi-88** [comment]
Yeah the changes look good, can you please look into  the failed DCO check for instructions to clear it. I dont think you have signed the commits properly you can check this [contribution guide](https://github.com/Avi-88/layer5/blob/master/CONTRIBUTING.md#prerequisites) for help ,thanks

**Pull request closed without merging**